### DISCUSSION TITLE
Help wanted: Lazy Load and LuaRocks Support

### DISCUSSION THREAD

**Discussion by 0oAstro** [wbthomason/packer.nvim#427] [Q&A] [answered]
Hey! I'm migrating from paq to packer for lazy loading feature. 

- Would you please help me by giving some docs and commands required to lazy load plugins? 
- How can I use Rocks with Packer? 
- Suppose I want to load neoformat (a plug-in to format code) only on save and not on startup, how can I do that?
- How can I run a command when plugin is installed? TreeSitter requires :TSUpdate on install.

**Comment by wbthomason** [comment, marked as answer]
Hi! I'd suggest having a look through the README if you haven't already - your questions should already all be answered there. In particular, have a look at https://github.com/wbthomason/packer.nvim#specifying-plugins, which describes how to specify plugins and lazy loaders (including all the features you mention). There are also some examples inhttps://github.com/wbthomason/packer.nvim#quickstart. Hope this helps!

**Reply by 0oAstro** [reply]
Yes this helped in solving my doubts.
